lib/main.dart

```diff
@@ -73,8 +73,9 @@ void main() async {
   await Firebase.initializeApp(options: DefaultFirebaseOptions.currentPlatform);
   await MobileAds.instance.initialize();
   final prefs = await StreamingSharedPreferences.instance;
+  final ttsModel = TtsModel.emptyConstructor();
```

Comment:
Does this actually work? The `TtsModel` should be initialized with `fromJson()` read from shared preferences.